source/documentation/common/install/snip-Enable_virt_module.adoc

```diff
@@ -14,15 +14,22 @@
 ----
 # dnf module enable virt:8.3
 ----
-* For {virt-product-shortname} 4.4.6 or later:
+* For {virt-product-shortname} 4.4.6 to 4.4.9:
 +
 [source,terminal]
 +
 ----
 # dnf module enable virt:av
 ----
+* For {virt-product-shortname} 4.4.10 and later:
```

Comment:
4.4SP1 (in RHV doc) or 4.5 in oVirt
4.4.10 still uses av

Reply:
Changes complete.